src/entities/students-entity.ts

```diff
@@ -0,0 +1,143 @@
+import {
+  Column,
+  Entity,
+  PrimaryGeneratedColumn,
+  OneToOne,
+  JoinColumn,
+  CreateDateColumn,
+} from 'typeorm';
+import { UsersEntity } from './users.entity';
+
+export enum ExpectedWorkType {
+  Static = 'Na miejscu',
+  CanMoveOut = 'Gotowość do przeprowadzki',
+  RemoteOnly = 'Wyłącznie zdalnie',
+  Hybrid = 'Hybrydowo',
+  Any = 'Bez znaczenia',
+}
+
+export enum ExpectedContractType {
+  UoPOnly = 'Tylko UoP',
+  B2B = 'Możliwe B2B',
+  UZorUOD = 'Możliwe UZ/UOD',
+  Any = 'Brak preferencji',
+}
+
+export enum GradingScale {
+  Score1 = 1,
+  Score2 = 2,
+  Score3 = 3,
+  Score4 = 4,
+  Score5 = 5,
+}
+
+export enum StudentStatus {
+  Available = 'Dostępny',
+  DuringRecruitment = 'W trakcie rozmowy',
+  Hired = 'Zatrudniony',
+}
+
+@Entity()
+export class StudentsEntity {
+  @PrimaryGeneratedColumn('uuid')
+  id: string;
+
+  @Column()
+  firstName: string;
+
+  @Column()
+  lastName: string;
+
+  @OneToOne(() => UsersEntity, {
+    onDelete: 'CASCADE',
+  })
+  @JoinColumn()
+  user: UsersEntity;
+
+  @Column({
+    type: 'varchar',
+    length: '9',
```

Comment:
A jak ktoś wpisze ze spacjami, z dywizami, z kierunkowym do Polski? Mamy to walidować na froncie i wysyłać Wam bez tych dodatków?

Reply:
dajcie info jak mają podać (ja osobiście wpisałbym poprostu 321321321 - 9 cyfr i tyle, nie mam pojęcia czy ktoś wpisuje specjalnie +48 a potem swój numer, można to wydłużyć nie widzę problemu).

Reply:
Może słusznie, podamy wymagany format i tyle. To nic nie zmieniaj.